infra/recipes/terraform/azure/environments/sandbox/main.tf

```diff
@@ -1,35 +1,82 @@
-resource "azurerm_resource_group" "openhouse_sandbox" {
-    name = var.resource_group_name
-    location = var.resource_group_location
+data "azurerm_resource_group" "openhouse_sandbox" {
```

Comment:
Add a descriptive comment on purpose of the sandbox environment in the beginning of the file

also briefly summarize all the modules/data/resources being spinned up. Goal for this documentation should be for layman to understand  e2e workflow.

Reply:
Done